src/rules/prefer-in-document.js

```diff
@@ -117,6 +122,12 @@ export const create = (context) => {
 
           // Remove any arguments in the matcher
           for (const argument of Array.from(matcherArguments)) {
+            const sourceCode = context.getSourceCode();
+            const token = sourceCode.getTokenAfter(argument);
+            if (token.value === "," && token.type === "Punctuator") {
+              // Remove commas if toHaveLength had more than one argument or a trailing comma
+              operations.push(fixer.replaceText(token, ""));
+            }
             operations.push(fixer.remove(argument));
           }
```

Comment:
A better fix for this should be to use the starting location of the first argument and then the ending location of the last argument - that way we don't have to actually care about the commas at all.

Reply:
You might me right, I thought about it also (couldn't tried since I don't know to working mechanism of eslint very well),
but I felt like comments would create problems if we handled it that way

Reply:
Problems with comments are not something we can really avoid - I'm happy to pick up getting the PR landed if you've not got the bandwidth :)

Reply:
that would be great, thanks 🙂

Reply:
ok so I was thinking about the handing for another situation (ESLint fixers are a bit mystic because of all the different ways code can be shaped 😂) - what you've got is good; it also turns out we're not handling trailing commas properly in `eslint-plugin-jest` either so thanks for being this to my attention!